src/transformers/models/llava_next/image_processing_llava_next.py

```diff
@@ -460,6 +568,7 @@ def preprocess(
         do_normalize: bool = None,
         image_mean: Optional[Union[float, List[float]]] = None,
         image_std: Optional[Union[float, List[float]]] = None,
+        do_pad: Optional[bool] = True,
```

Comment:
Forgot to mention - this should be added and documented in the __init__ too

Reply:
Okay, will add it and should be good to go. Also I'll commit a run-slow message, in case we should check everything is okay